pizza.json

```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "position": 1,
+    "name": "Traditional",
+    "price": 19,
+    "quantity": 0
+  },

```

Comment:
As I told you, you can rework it the following way:
```json
{
	1: {
		"name": "Traditional",
		"price": 19,
		"ingridients": ["bacon", "tomatos", "magic"]
	},
	2: {
		"name": "Margherita",
		"price": 22,
		"ingridients": ["margaritas", "onions", "sorcery"]
	}
}
```

After loading this JSON to the variable `pizzas` that will allow you to do the following checks:
`if pizzas.get(usrs_choice):` where `usrs_choice` is a position from the menu.

Reply:
Need to try it in the test file first. This change can ruin all of my efforts...

Reply:
But this rework could make my code better looking))

Reply:
BTW! "Ingredients feature" will be my next step. So, probably, I'll use your advice with applying this feature!